app/src/main/res/values/dimens.xml

```diff
@@ -214,157 +50,350 @@
+  <dimen name="content_item_question_view_margin_start">32dp</dimen>
+  <dimen name="previous_responses_item_question_split_view_margin_end">80dp</dimen>
+  <dimen name="previous_responses_item_question_split_view_margin_start">80dp</dimen>
+  <dimen name="continue_item_question_split_view_margin_top">72dp</dimen>
+  <dimen name="submit_button_item_question_split_view_margin_top">72dp</dimen>
+  <dimen name="submitted_answer_question_split_view_margin_top">24dp</dimen>
+  <dimen name="submitted_answer_question_view_margin_top">24dp</dimen>
+  <dimen name="replay_button_item_question_view_padding_end">24dp</dimen>
+  <dimen name="replay_button_item_question_view_padding_start">24dp</dimen>
+  <dimen name="submitted_answer_question_split_view_margin_end_no_extra_interaction_answer">24dp</dimen>
+  <dimen name="submitted_answer_question_split_view_margin_top_no_extra_interaction_answer">24dp</dimen>
+  <dimen name="submit_button_item_question_view_padding_end">24dp</dimen>
+  <dimen name="submit_button_item_question_view_padding_start">24dp</dimen>
+  <dimen name="return_to_topic_button_question_view_padding_end">24dp</dimen>
+  <dimen name="return_to_topic_button_question_view_padding_start">24dp</dimen>
+  <dimen name="continue_item_question_view_margin_top">72dp</dimen>
+  <dimen name="continue_navigation_item_question_split_view_margin_top">72dp</dimen>
+  <dimen name="next_button_item_question_split_view_margin_top">72dp</dimen>
+  <dimen name="previous_button_item_question_split_view_margin_top">72dp</dimen>
+  <dimen name="replay_button_item_question_split_view_margin_top">72dp</dimen>
   <dimen name="submitted_answer_question_split_view_margin_end_extra_interaction_answer">64dp</dimen>
   <dimen name="submitted_answer_question_split_view_margin_start_extra_interaction_answer">64dp</dimen>
+  <dimen name="content_item_question_split_view_margin_bottom">12dp</dimen>
+  <dimen name="content_item_question_view_margin_bottom">12dp</dimen>
+  <dimen name="replay_button_item_question_view_padding_bottom">12dp</dimen>
+  <dimen name="continue_interaction_item_question_view_margin_end">24dp</dimen>
+  <dimen name="continue_interaction_item_question_view_margin_start">24dp</dimen>
+  <dimen name="continue_navigation_item_question_view_margin_end">24dp</dimen>
+  <dimen name="continue_navigation_item_question_view_margin_start">24dp</dimen>
+  <dimen name="drag_drop_item_question_view_margin_top">24dp</dimen>
+  <dimen name="feedback_item_question_view_margin_top">24dp</dimen>
+  <dimen name="next_button_question_view_padding_end">24dp</dimen>
 
-  <dimen name="story_chapter_card_playable_state_elevation">4dp</dimen>
-  <dimen name="ongoing_topic_card_elevation">4dp</dimen>
-  <dimen name="story_chapter_card_not_playable_state_elevation">2dp</dimen>
-  <dimen name="profile_chooser_padding_bottom_profile_already_added">224dp</dimen>
-
+  <!--Exploration-->
+  <dimen name="previous_button_item_exploration_split_view_margin_start">48dp</dimen>
```

Comment:
Break these into multiple parts like 
`Previous Button`
`Previous Responses`
`Content Item`
`Submit Button`
`Submitted Answer`
`Return to Topic`
`Replay Button`
`Feedback Item`
`Continue Interaction Item`
`Continue Navigation Item`

etc. The main criteria is if there is an `xml` file then it should have its separate section like above names.

Reply:
Do i have to subdivide inside every format like  <!-- Exploration > <!--Previous Button><dimen name="previous"...../> <!--Previous response>......

Reply:
Just update your branch with latest develop and there you will see the correct format at the bottom that I have followed for `content-item`